packages/react/src/components/Dropdown/Dropdown.js

```diff
@@ -44,12 +44,14 @@ function Dropdown({
   invalidText,
   initialSelectedItem,
   selectedItem: controlledSelectedItem,
+  downshiftProps,
 }) {
   const selectProps = {
     items,
     itemToString,
     initialSelectedItem,
     onSelectedItemChange,
+    ...downshiftProps,
```

Comment:
I think this should be the first thing in the object to match previous behavior e.g.

```js
const selectProps = {
  ...downshiftProps,
  items,
  // ...
}
```